src/server.js

```diff
@@ -79,7 +78,7 @@ module.exports = function (classes) {
       _checkAuth: function (req) {
         var self = this;
 
-        if (self.authHandler) {
+        if (self.authType !== Authorization.NONE) {
```

Comment:
I think this was the part that I did not understand fully and needed to be discussed furthermore. 
If we use authType as a function parameter, how do we switch authorization types after initializing more than one ?